plugins/attachment-resources/src/utils.ts

```diff
@@ -84,3 +86,19 @@ export function getType (type: string): 'image' | 'video' | 'audio' | 'pdf' | 'o
 
   return 'other'
 }
+
+export async function moveAttachments (doc: Doc, space: Ref<Space>): Promise<void> {
+  const client = getClient()
+  const hierarchy = client.getHierarchy()
+  const attributes = hierarchy.getAllAttributes(doc._class)
+  for (const [name, attribute] of attributes) {
```

Comment:
AttachedDoc is not an attachment* plugin concept, you should move this logic into package/core

Reply:
Not sure but moved to `TxOperations`
Should we update only `space` for AttachedDoc?